utils/database.js

```diff
@@ -69,11 +86,19 @@ const queryDatabase = function(database, sql, values) {
     let connection;
     return poolPromise.then((p) => {
         return p.getConnection();
-    }).then((conn) => {
+    }).then(async (conn) => {
         connection = conn;
         conn.changeUser({database: database});
-        console.log(sql);
-        return conn.query(sql, values);
+
```

Comment:
is the sql no longer worth logging?

Reply:
Now log the sql after it executes, so as to also include the latency.

If the query fails then we won't have logged it. Can add an error handler.